crates/matrix-sdk-base/src/rooms/normal.rs

```diff
@@ -1778,6 +1823,65 @@ mod tests {
         );
     }
 
+    #[async_test]
+    async fn test_save_bad_joinrules() {
```

Comment:
This test is currently not testing what it's supposed to, because it runs in-memory instead of on sqlite or indexeddb. How can I change that?

Reply:
Did you address this comment? ^